src/app.js

```diff
@@ -0,0 +1,142 @@
+import axios from 'axios';
+
+// Reference naar HTML element
+const countryInfo = document.getElementById('country-info');
+
+// Functie die een object en een string verwacht en een boolean teuggeeft om alle alternatieve namen van landen te zoeken
+function checkAltNames(object, input) {
+    if (input === '') {
+        return false;
+    }
+    if (object.name.toLowerCase() === input.toLowerCase()) {
+        return true;
+    }
+    if (object.nativeName.toLowerCase() === input.toLowerCase()) {
+        return true;
+    }
+    if (object.altSpellings === undefined) {
+        return false;
+    }
+    for (let i = 0; i < object.altSpellings.length; i++) {
+        if (object.altSpellings[i].toLowerCase() === input.toLowerCase()) {
+            return true;
+        }
+    }
+    return false;
+}
+
+// Functie die een array verwacht en een string terug geeft met de valuta's die gebruikt kunnen worden
+function valuta(array) {
+    if (array.length === 1) {
+        return array[0].name;
+    }
+    if (array.length === 2) {
+        return array[0].name + `'s and ${array[1].name}`;
+    }
+}
+
+// Functie die een array verwacht en een string terug geeft met de gesproken talen
+function language(array) {
+    let text = '';
+    for (let i = 0; i < array.length; i++) {
+        if (i === 0) {
+            text += `They speak ${array[i].name}`;
+        }
+        if (i > 0 && i < array.length -1) {
+            text += `, ${array[i].name}`;
+        }
+        if (i > 0 && i === array.length - 1) {
+            text += ` and ${array[i].name}`;
+        }
+    }
+    return text;
```

Comment:
Je zou dit nog iets kunnen versimpelen:
```suggestion
    let text = `They speak ${array[0].name}`;
    for (let i = 1; i < array.length; i++) {
        if (i < array.length -1) {
            text += `, ${array[i].name}`;
        } else {
            text += ` and ${array[i].name}`;
        }
    }
    return text;
```